buildSrc/src/main/java/org/elasticsearch/gradle/testclusters/RunTask.java

```diff
@@ -0,0 +1,69 @@
+package org.elasticsearch.gradle.testclusters;
+
+import org.gradle.api.logging.Logger;
+import org.gradle.api.logging.Logging;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+import org.gradle.api.tasks.options.Option;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.HashSet;
+import java.util.Set;
+
+public class RunTask extends DefaultTestClustersTask {
+
+    private static final Logger logger = Logging.getLogger(RunTask.class);
+
+    private Boolean debug = false;
+
+    @Option(
+        option = "debug-jvm",
+        description = "Enable debugging configuration, to allow attaching a debugger to elasticsearch."
+    )
+    public void setDebug(boolean enabled) {
+        this.debug = debug;
+    }
+
+    @Input
+    public Boolean getDebug() {
+        return debug;
+    }
+
+    @Override
+    public void beforeStart() {
+        int port = 8000;
+        for (ElasticsearchCluster cluster : getClusters()) {
+            for (ElasticsearchNode node : cluster.getNodes()) {
+                node.setHttpPort("9200");
```

Comment:
Won't this fail to bind for clusters w/ > 1 node? We seem to be incrementing the debug port, should we do the same for the HTTP and transport ports?

Reply:
Indeed, I added port increment.